bedrock/firefox/templates/firefox/compare/chrome.html

```diff
@@ -201,42 +203,45 @@ <h2>{{ _('<strong>Utility</strong>') }}</h2>
           </tbody>
         </table>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         In terms of features, both Firefox and Chrome offer a large library of extensions and plug-ins, with Chrome’s catalog vastly outnumbering any other browser while nicely integrating with other Google services, like Gmail and Google Docs.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         Although not as extensive as Chrome’s add-on library, Firefox, as open-source software with a huge number of active contributors, also features an incredible number of useful extensions.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         Firefox also has a sync feature to see your open and recent tabs, browsing history, and bookmarks across all your devices.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
+        {# L10n: "gets the nod" here means Chrome is the winner. It's an idiom that indicates making a selection, like someone nodding their head in approval. #}
         {% trans %}
         While Chrome gets the nod with add-ons and extensions, Firefox has a nicely curated set of built-in features, such as the incredibly handy screen capture tool, and reading mode feature which strips away everything from the page except the text from the article you’re reading.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         If having tons of open tabs is your thing, then it really comes down to your UI preference. Firefox features a horizontal scroll on all your open tabs rather than shrinking them smaller and smaller with each new one. Google Chrome prefers to shrink them down so just the favicon is visible. The only problem with this is when you have multiple tabs open from the same website, so you see the same favicon across your tabs.
         {% endtrans %}
       </p>
 
-      <p class="compare-content">
+      <p>
         {% trans %}
         As for customization, our fans will tell you one of the things they love most about our browser is its ability to allow you to move and arrange a majority of the UI elements to best suit your needs. Chrome allows you to hide certain UI elements but there’s not much allowance, if any, for moving things around based on your preferences. However, it should be noted that both Chrome and Firefox make it pretty easy to change your browser’s appearance and theme.
         {% endtrans %}
       </p>
 
+      {% include '/firefox/compare/includes/download-menu-list.html' %}
```

Comment:
Only the first occurence of this menu in a page seems to work? The others don't seem to open when clicked.

Reply:
I see the problem. I didn't realize the details JS uses an ID to know which menu to open and the include has duplicate IDs (which is a problem anyway, but now it's also breaking the JS). So when you click the second menu it's actually opening the _previous_ one further up the page.

I started to make the ID a variable to pass to the include, but realized that because Stephanie is brilliant and already thought of this, the JS will automatically assign a unique ID itself if ID. So just removing the ID from the include also fixes it.